autowebcompat/crawler.py

```diff
@@ -0,0 +1,18 @@
+from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException
```

Comment:
@marco-c What do you suggest for the file this file's name? or it is good to move this function to utilis.py

Reply:
I think `crawler.py` is OK.